src/index.ts

```diff
@@ -1,8 +1,8 @@
 import Serverless from 'serverless';
-import Plugin from 'serverless/classes/Plugin';
+import type Plugin from 'serverless/classes/Plugin';
 import { PluginConfig } from './PluginConfig';
 
-class ServerlessDisableLocalPlugin {
+export default class ServerlessDisableLocalPlugin {
```

Comment:
When I first built it, the `export default` didn't work for me with API. Could be combinations of those. Could be other reasons which I didn't get a chance to look further hence I choose to use commonjs export.

 Quickly pulled your branch and can confirm that it doesn't work with API. Screenshot below.

![image](https://github.com/beforeyoubid/serverless-disable-local/assets/64051705/8abc7113-e129-4cd4-b8e6-143fa7433a73)

Reply:
The output package does the correct output by exporting as a default - 
![image](https://github.com/beforeyoubid/serverless-disable-local/assets/64051705/12985bc4-509c-468c-b6d8-cff0309d9f67)